docs/explanations.rst

```diff
@@ -5,11 +5,49 @@ Our explanatory and conceptual guides are written to provide a better
 understanding of how the Ubuntu Pro Client (``pro``) works. They enable you
 to expand your knowledge and become better at using and configuring ``pro``.
 
-Explanation
-===========
+Messaging
+=========
+
+Here you'll find details about APT and MOTD messages -- what they are, when
```

Comment:
We could specify here that the section has information about the *pro client related* APT and MOTD messages - as those are wider concepts and there are messages there that we do not control

Reply:
That's a great point - changed it to reflect your suggestion!